demos/README.md

```diff
@@ -36,16 +36,13 @@ ovms_demo_python_llama
 ovms_demo_python_seq2seq
 ovms_demo_python_stable_diffusion
 ovms_string_output_model_demo
+ovms_demos_continous_batching
 ```
 
 OpenVINO Model Server demos have been created to showcase the usage of the model server as well as demonstrate it’s capabilities.
 ### Check Out New Generative AI Demos
- - [Text Generation](python_demos/llm_text_generation/README.md)
- - [Stable Diffusion](python_demos/stable_diffusion/README.md)
+ - [Text Generation with continous batching](continous_batching/README.md)
 
-**See also new multi-modal demo:**
-- [CLIP image classification](python_demos/clip_image_classification/README.md)
```

Comment:
Do we want to remove all demos from previous release from this section? Stable diffusion and multi modal stuff can still be done with Python.